pkg/restore/restore.go

```diff
@@ -632,6 +632,8 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 			continue
 		}
 
+		fromBackup := obj.DeepCopy()
```

Comment:
Mind adding a comment above with a brief blurb about why we're doing the copy?

Reply:
comment added